packages/@aws-cdk/aws-rds/test/proxy.integ.snapshot/aws-cdk-rds-proxy.template.json

```diff
@@ -498,7 +498,7 @@
      "Ref": "dbInstanceSubnetGroupD062EC9E"
     },
     "Engine": "postgres",
-    "EngineVersion": "11.15",
+    "EngineVersion": "11.16",
```

Comment:
@michal-draftwise I'm assuming that you just manually edited this file to fix the snapshot failure? I think in this case it is fine to just leave the integ test at `VER_11_15`. I don't think it is necessary to have an integration test for _every_ minor version.